wpilib/wpilib/drive/robotdrivebase.py

```diff
@@ -52,24 +52,23 @@ def __init__(self):
         self.setSafetyEnabled(True)
         self.setName("RobotDriveBase")
 
-    def setDeadband(self, deadband):
-        """Change the value for deadband scaling. The default value is `.02`. Values smaller than the
-         deadband are set to 0, while values larger than the deadband are scaled from 0.0 to 1.0. See
-         :meth:`applyDeadband()`
-
-         :param deadband: The deadband to set
-         :type deadband: float
-         """
+    def setDeadband(self, deadband: float):
```

Comment:
Why here and not everywhere else

Reply:
I don't see anywhere else in this file that has type information. Too lazy to add in the missing types.